### PR TITLE
feat: add personal identifier to feedback form

### PR DESCRIPTION
**Pull request by djaustin**

### CODE REVIEW

**Comment by djaustin** [comment]
test

**Comment by djaustin** [comment]
test

**Comment by djaustin** [comment]
test

**Review by annabelkimber — APPROVED** [review]

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
-  const { handleSubmit, control } = useForm<FormInputs>();
+  const {
+    handleSubmit,
+    control,
+    register,
+    formState: { errors },
+  } = useForm<FormInputs>();
+  const [contactUser, setContactUser] = useState(false);
   const sendFeedback: SubmitHandler<FormInputs> = (data) => {
+    console.log(data);
```
Comment (tigs1995) [inline src/pages/index.tsx:30]: Remove

**Review by tigs1995 — APPROVED** [review]

**Pull request merged**